paper.md

```diff
@@ -2,11 +2,77 @@
 title: "Popper 2.0: A Container-Native Workflow Execution Engine For Testing Complex Applications and Reproducing Scientific Explorations"
 author: Ivo Jimenez, Jayjeet Chakraborty, Arshul Mansoori, Quincy Wofford and Carlos Maltzahn
 abstract: |
- Abstract
+ Software containers allow users to "bring their own environment" to 
+ shared computing platforms, reducing the friction between system 
+ administrators and their users. In recent years, multiple container 
+ runtimes have arisen, each addressing distinct needs (e.g. 
+ Singularity, Podman, rkt, among others), and an ongoing effort from 
+ the Linux Foundation (Open Container Initiative) is standardizing the 
+ specification of Linux container runtimes. While containers solve a 
+ big part of the "dependency hell" problem, there are scenarios where 
+ multi-container workflows are not fully addressed by existing 
+ runtimes or workflow engines. Current alternatives require a full 
+ scheduler (e.g. Kubernetes), a scientific workflow engine (e.g. 
+ Pegasus), or are constrained in the type of logic that users can 
+ express (e.g. Docker-compose). Ideally, users should be able to 
+ express workflows with the same user-friendliness and portability of 
+ `Dockerfile`s (write once, run anywhere). In this article, we 
+ introduce "Popper 2.0" a container-native workflow execution engine 
+ that allows users to express complex workflows similarly to how they 
+ do it in other scientific workflow languages, but with the advantage 
+ of running in container runtimes, bringing portability and ease of 
+ use to HPC scenarios. Popper 2.0 cleanly separates the three main 
+ concerns that are common in HPC scenarios: experimentation logic, 
+ environment preparation, and system configuration. To exemplify the 
+ suitability of the tool, we present three different case studies 
+ where we take examples from Machine Learning and HPC experiments
+ and turn them into Popper workflows.
 ---
 
 # Introduction {#sec:intro}
 
+Scientists and researchers often leave experimental artifacts
```

Comment:
these first paragraphs are missing references, as discussed previously. The references need to provide proof or show that this is a problem that has been discussed/addressed somewhere else. For example, the sentence:

> Scientists and researchers often leave experimental artifacts [leave them where?] like scripts, ..., etc [missing dot on etc] after completing with their work [sic]

this needs to be accompanied with a references that proves this claim

Reply:
@ivotron  what did you mean by `sic`?

Reply:
that "completing with their work" needs to be fixed.  see [here](https://en.wikipedia.org/wiki/Sic) for more

Reply:
Oh. I should have googled it.